tests/foreman/api/test_repositories.py

```diff
@@ -190,3 +190,29 @@ def test_positive_sync_kickstart_repo(self, module_entitlement_manifest_org, tar
     rh_repo = rh_repo.read()
     assert rh_repo.content_counts['package_group'] > 0
     assert rh_repo.content_counts['rpm'] > 0
+
+
+@pytest.mark.tier1
+def test_positive_multiple_orgs_with_same_repo(target_sat):
+    """Test that multiple organizations with the same repository synced have matching metadata
+
+    :id: 39cff8ea-969d-4b8f-9fb4-33b1ba768ff2
+
+    :Steps:
+        1. Create multiple organizations
+        2. Sync the same repository to each organization
+        3. Assert that each repository from each organization contain the same content counts
+
+    :expectedresults: Each repository in each organziation should have the same content counts
+    """
+    repos = []
+    orgs = [target_sat.api.Organization().create() for _ in range(3)]
+    for org in orgs:
+        product = target_sat.api.Product(organization=org).create()
+        repo = target_sat.api.Repository(
+            content_type='yum', product=product, url=settings.repos.module_stream_1.url
```

Comment:
Non-blocking:
We can extend this to all content types of repositories

Reply:
@adarshdubey-star The repo I am syncing in this case is a yum only repo. parameterized content_type would not be possible unless we test this with multiple custom repos each a different type. that may qualify for a separate test. I can sync with support and find out.